mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx

```diff
@@ -125,7 +125,7 @@ export function DexScreen (): JSX.Element {
     }, 500),
     [activeTab, pairs]
   )
-  const [activeButtonGroup, setActiveButtonGroup] = useState<ButtonGroupTabKey>(ButtonGroupTabKey.AllPairs)
+  const [activeButtonGroup] = useState<ButtonGroupTabKey>(ButtonGroupTabKey.AllPairs)
```

Comment:
what is the point of using `state` if it is just a constant?

Reply:
Because there's no filtering on this PR. will consolidate it with Harsh's PR